### ISSUE TITLE
Replace inline SVGs with IMG elements in Projects' Resources section

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#4291]
### Overview

SVG files instantiated using an inline `<svg>` element do not contain alt text.  To ensure accessibility, we will add the `aria-label `attribute to the surrounding `<div>`. 
  
###  Items

Each Project page has a Resources section which is rendered from https://github.com/hackforla/website/blob/gh-pages/_includes/resource-card.html.  

- [x] Understand how the various images are rendered, based on the liquid variable `name`.  Note the condition`{% if image %}`within the enclosing div (with `class="resource-img"`).

- [x] Only for the case in which the image is rendered using `<svg>`, add an `aria-label` attribute to the enclosing div, with value `{{ name }}`

### Resources/Instructions

- sample project page: https://www.hackforla.org/projects/311-data
- [Wiki "How to make SVGs and other images WCAG compliant"](https://github.com/hackforla/website/wiki/How-to-make-SVGs-and-other-images-WCAG-compliant)
- [_layouts/project.html](https://github.com/hackforla/website/blob/d3b10fadfefa646ade8bdd5e3de6b89fa1ad466b/_layouts/project.html)
- [_includes/resource-card.html](https://github.com/hackforla/website/blob/d3b10fadfefa646ade8bdd5e3de6b89fa1ad466b/_includes/resource-card.html)
- [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [Using alt attributes on img elements](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Logo image within link text](https://www.w3.org/WAI/tutorials/images/functional/#example-2-logo-image-within-link-text)
- This issue was created as a result of #4242

**Comment by roslynwythe** [comment]
@ExperimentsInHonesty please confirm that the alt text instructions (under Resources/Instructions) are correct, based on your comment https://github.com/hackforla/website/issues/2626#issuecomment-1297790151

- Also the title of  #4242 mentions a wiki.  Jessica created this wiki page which describes the situation very well so I assume that is sufficient: https://github.com/hackforla/website/wiki/How-to-make-SVGs-and-other-images-WCAG-compliant

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe Thanks for the notes.  I have posted something in the ER about Jessica's wiki page.  This issue is good to go.

**Comment by mattmalane** [comment]
**Availability:** M-F 3PM - 6PM Pacific
**ETA:** Wed, March 29 6PM Pacific

**Comment by mattmalane** [comment]
Hi @roslynwythe, I've looked over this issue and have read Jessica's wiki page on when to use either an `img` tag with an `alt` property or  a `div` in combination with an `ARIA` label and inline `SVG`.

**Problem**

Currently the `svg` files that are being "included" exist in the `_includes/svg` directory like they should for the current implementation. In order for me to replace the inline svg elements with `img` tag w/  the appropriate `src` attribute, the svg images from `_includes/svg` needs to be stored in a directory like `assets/images` or a subfolder within it (I manually looked through `assets/images` for the appropriate svg files, but didn't find any. I could have overlooked though). Looks like when Jekyll builds the site, the `_includes` folder gets ignored, which hosts the `svg` files.

**Possible Solutions**

I could be missing something here due to oversight, please let me know if I'm not understanding the problem completely. From reading the wiki and some more research here are some solutions:

1.  store the following images currently located in `_includes/svg` in `assets/images` or a subfolder:

  - `icon-github-color.svg`
  - `icon-slack-color.svg`
  - `logo-hfla-small.svg`
  - `_iconicons_svg_ios-link.svg`
  
    This should allow us to use `img` tags wth `src` attributes that can locate the `svg` images.
 
2. [Wrap the current implementation of inline svg elements with a div and include ARIA label.](https://github.com/hackforla/website/wiki/How-to-make-SVGs-and-other-images-WCAG-compliant#use-an-aria-label-in-the-div-html-tag-wrapping-the-svg)

**Comment by roslynwythe** [comment]
Hi @mattmalane thank you for taking this issue and your analysis which is absolutely correct.  I appreciate your clear presentation of the problem and possible solutions.   

I'm not sure if it is safe to move those .svg files.  Let me do a search to see if they are referenced elsewhere.  I can take a look after tonight's onboard meeting.  Or you could check it out.  If the svgs are included in other places, we could make a copy of the files and place them in assets/images.     

You're correct about the option of wrapping the `<svg>` tag in a `<div>` with an aria-label.  Do you have an opinion about which is preferable in this scenario?

I won't be at tonight's meeting, since there is an onboarding meeting at the same time.  You're welcome to raise this at tonight's meeting if you want, but I'm going to remove the Blocker label so that other leads don't have to take time to look at it.

**Comment by roslynwythe** [comment]
Hi @mattmalane please see the updates to the Action Items.  If they are not clear or don't seem correct, please let me know.

**Comment by mattmalane** [comment]
@roslynwythe Awesome, thanks!
